README.rst

```diff
@@ -14,8 +14,8 @@ Please read the `Bulebule documentation`_ for a bit more detailed introduction.
 Acknowledgments
 ---------------
 
-We truly feel in debt with some people who helped us a lot. This project was
-much faster to develop and enjoyable thanks to:
+We truly feel indebted to some people who helped us a lot. This project was
+much faster to develop and enjoyable. Thanks to:
```

Comment:
I asked a native American and they suggested "We truly feel indebted to those who helped us. This project was much faster to develop and enjoyable thanks to:"

Mind changing it? :blush: 

Also, for consistency with our commit subject style, I would suggest "Fix some grammatical errors in README file"